include/nuttx/mtd/mtd.h

```diff
@@ -820,6 +820,53 @@ int dhara_initialize_by_path(FAR const char *path,
                              FAR struct mtd_dev_s *mtd);
 #endif
 
+/****************************************************************************
+ * Name: nvblk_initialize
+ *
+ * Description:
+ *   Initialize to provide a block driver wrapper around an MTD interface
+ *
+ * Input Parameters:
+ *   minor - The minor device number.  The MTD block device will be
+ *           registered as as /dev/mtdblockN where N is the minor number.
+ *   mtd   - The MTD device that supports the FLASH interface.
+ *   lbs  - The logical blocksize (size of the nvblk blocks).
+ *   iobs - The input output blocksize (multiple of lbs).
+ *   speb - The number of spare erase blocks.
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_MTD_NVBLK
+int nvblk_initialize(int minor,
+                     FAR struct mtd_dev_s *mtd,
+                     uint32_t lbs,
+                     uint32_t iobs,
+                     uint32_t speb);
+#endif
+
+/****************************************************************************
+ * Name: nvblk_initialize_by_path
+ *
+ * Description:
+ *   Initialize to provide a block driver wrapper around an MTD interface
+ *
+ * Input Parameters:
+ *   path - The block device path.
+ *   mtd  - The MTD device that supports the FLASH interface.
+ *   lbs  - The logical blocksize (size of the nvblk blocks).
+ *   iobs - The input output blocksize (multiple of lbs).
+ *   speb - The number of spare erase blocks.
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_MTD_NVBLK
+int nvblk_initialize_by_path(FAR const char *path,
```

Comment:
```suggestion
int nvblk_initialize(FAR const char *path,
```
and remove line 840, like https://github.com/apache/nuttx/pull/16793

Reply:
Done